### PR TITLE
Vita - Bestiar

### PR DESCRIPTION
**Pull request by vitvrs**
Bestiář se zobrazuje na .../bestar/

### CODE REVIEW

**Comment by just-paja** [comment]
@vitvrs, it does not build. Please fix.

**Comment by vitvrs** [comment]
> @vitvrs, it does not build. Please fix.

@just-paja fixed ;)

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "root",
   "private": true,
   "dependencies": {
+    "download": "^8.0.0",
```
Comment (just-paja) [inline package.json:5]: We shouldn't pull in stuff we don't need. Please remove this extra dependency

File: packages/fantasion-web/components/contact.module.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import '../styles/variables.scss';
```
Comment (just-paja) [inline packages/fantasion-web/components/contact.module.scss:1]: This CSS module is empty. Please remove it, so we don't pollute the codebase.

File: packages/fantasion-web/components/monsters.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <MonsterListItem monster={monster} />
+      </Col>
+    ))}
+  </Row>
+)
+
+export const MonsterDetail = ({ monster }) => (
+  <Article
+    title={monster.title}
+    // subTitle={monster.species}
```
Comment (just-paja) [inline packages/fantasion-web/components/monsters.js:79]: Dead code. Please remove.

File: packages/fantasion-web/components/monsters.module.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
+@keyframes appear {
```
Comment (just-paja) [inline packages/fantasion-web/components/monsters.module.scss:1]: Let's keep this, but remember that the code needs to be deduplicated.

File: packages/fantasion-web/pages/contact.js [hunk 5]
```diff
@@ -5,8 +5,9 @@ import Row from 'react-bootstrap/Row'
 
 import { asPage, MetaPage } from '../components/meta'
 import { ContactCard } from '../components/ContactCard'
+// import { ContactForm } from '../components/contact'
```
Comment (just-paja) [inline packages/fantasion-web/pages/contact.js:8]: Dead code. Please remove.

File: packages/fantasion-web/pages/contact.js [hunk 6]
```diff
@@ -5,8 +5,9 @@ import Row from 'react-bootstrap/Row'
...
 import { GenericPage } from '../components/layout'
-import { GeneralNewsletterForm } from '../components/GeneralNewsletterForm'
+// import { Heading } from '../components/media'
```
Comment (just-paja) [inline packages/fantasion-web/pages/contact.js:10]: Dead code. Please remove.

File: packages/fantasion-web/pages/contact.js [hunk 7]
```diff
@@ -27,7 +28,8 @@ const Contacts = () => {
             <ContactCard />
           </Col>
           <Col md={6}>
-            <GeneralNewsletterForm className="mt-3" />
+            {/* <Heading relativeLevel={2}>{t('contact-form')}</Heading>
+            <ContactForm /> */}
```
Comment (just-paja) [inline packages/fantasion-web/pages/contact.js:32]: Dead code. Please remove.

**Review by just-paja — CHANGES_REQUESTED** [review]
Looks fine, just clean the code please.

I noticed the ContactForm component is unused at the moment and assume that's work in progress. Acceptable to merge given project's status.

**Review by just-paja — APPROVED** [review]

**Pull request merged**